invenio_app_ils/records/resolver/loan.py

```diff
@@ -41,9 +41,7 @@ def patron_resolver(loan_pid):
         return {}
 
     try:
-        p = Patron.get_patron(patron_pid)
-        patron = p.dumps()
-        patron.pop("$schema", None)
+        patron = Patron.get_patron(patron_pid).dumps_loader()
```

Comment:
Can we do the same as above?
```
from flask import current_app

    def get_patron(patron_pid):
        """Return the Patron record."""
        patron = current_app_ils_extension.patron_cls.get_patron(patron_pid)
        return patron.dumps_loader()
```

and same for `item = Item.get_record_by_pid(item_pid)`